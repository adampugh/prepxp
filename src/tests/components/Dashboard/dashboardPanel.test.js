import React from "react";
import { shallow } from "enzyme";

import DashboardPanel from "../../../components/Dashboard/dashboardPanel";
import user from "../../fixtures/userLists";

describe("<dashboardPanel />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<DashboardPanel 
            lists={user.lists}
            totalAnsweredQuestions={4}
            totalQuestions={10} />);
        expect(wrapper).toMatchSnapshot();
    });
});