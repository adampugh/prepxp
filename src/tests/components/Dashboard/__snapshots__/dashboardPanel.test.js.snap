// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<dashboardPanel /> should match snapshot 1`] = `
<div
  className="dashboard__grid__panel"
>
  <div
    className="dashboard__grid__panel__box"
  >
    <div>
      <h2>
        Questions Answered
      </h2>
      <t
        background={false}
        backgroundPadding={10}
        classForPercentage={null}
        className=""
        classes={
          Object {
            "background": "CircularProgressbar-background",
            "path": "CircularProgressbar-path",
            "root": "CircularProgressbar",
            "text": "CircularProgressbar-text",
            "trail": "CircularProgressbar-trail",
          }
        }
        counterClockwise={false}
        initialAnimation={true}
        percentage={100}
        strokeWidth={8}
        styles={
          Object {
            "background": Object {},
            "path": Object {},
            "root": Object {},
            "text": Object {},
            "trail": Object {},
          }
        }
        textForPercentage={[Function]}
      />
      <p>
        3 / 3
      </p>
    </div>
    <div>
      <h2>
        Lists Completed
      </h2>
      <t
        background={false}
        backgroundPadding={10}
        classForPercentage={null}
        className=""
        classes={
          Object {
            "background": "CircularProgressbar-background",
            "path": "CircularProgressbar-path",
            "root": "CircularProgressbar",
            "text": "CircularProgressbar-text",
            "trail": "CircularProgressbar-trail",
          }
        }
        counterClockwise={false}
        initialAnimation={true}
        percentage={0}
        strokeWidth={8}
        styles={
          Object {
            "background": Object {},
            "path": Object {},
            "root": Object {},
            "text": Object {},
            "trail": Object {},
          }
        }
        textForPercentage={[Function]}
      />
      <p>
        0 / 2
      </p>
    </div>
    <div
      className="dashboard__grid__panel__box__mostCompleted"
    >
      <h2>
        Most Completed
      </h2>
      <p
        key="0JavaScript Developer"
      >
        1. JavaScript Developer 
        <span>
          0
        </span>
      </p>
      <p
        key="1PHP Developer"
      >
        2. PHP Developer 
        <span>
          0
        </span>
      </p>
    </div>
  </div>
</div>
`;
